api/v1alpha1/release_types_test.go

```diff
@@ -17,437 +17,315 @@ limitations under the License.
 package v1alpha1
 
 import (
-	"time"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	. "github.com/onsi/gomega/gstruct"
-
-	applicationapiv1alpha1 "github.com/redhat-appstudio/application-api/api/v1alpha1"
+	"github.com/redhat-appstudio/release-service/conditions"
 	"k8s.io/apimachinery/pkg/api/meta"
+	"time"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type conditionValues struct {
-	status  metav1.ConditionStatus
-	reason  ReleaseReason
-	message string
-}
-
 var _ = Describe("Release type", func() {
-
-	var r *Release
+	var release *Release
```

Comment:
I'm not saying revert this change because I am reviewing it as is and both are fine, but I don't think such a simple change like changing `r` to `release` in a file with this scope gives us any benefit besides making the review process longer because the diff is bigger

Reply:
I rewrote the test to simplify it and I took the chance to fix a variable with a wrong name. Those kind of names are reserved for receivers, not for variables. Variable names should be meaningful.